utils/ccdatofhir/Package.md

```diff
@@ -0,0 +1,15 @@
+# C-CDA to FHIR Package
+
+## Package Overview
+
+Package containing C-CDA to FHIR pre-built mapping functionalities. 
+
+This package implements the C-CDA to FHIR implementation guide (http://build.fhir.org/ig/HL7/ccda-on-fhir/CF-index.htmlr/datatype_maps.html). 
+
+### Supported C-CDA to FHIR Mappings
+1) C-CDA Allergy Intolerance Observation to FHIR Allergy Intolerance
+2) C-CDA Immunization Activity to FHIR Immunization
+3) C-CDA Medication Activity to FHIR Medication
+4) C-CDA US Realm Heade to FHIR Patient
```

Comment:
```suggestion
4) C-CDA US Realm Header to FHIR Patient
```

Reply:
Fixed in https://github.com/ballerina-platform/module-ballerinax-health.fhir.r4/commit/bfbad6e342e0df400c6fffecc9682cf0f99e19b7